src/GovUk.Education.ExploreEducationStatistics.Public.Data.Model/DataSetVersionMapping.cs

```diff
@@ -38,52 +38,27 @@ public void Configure(EntityTypeBuilder<DataSetVersionMapping> builder)
                 .HasDatabaseName("IX_DataSetVersionMappings_SourceDataSetVersionId")
                 .IsUnique();
 
-            builder.HasIndex(mapping => new {mapping.TargetDataSetVersionId})
+            builder.HasIndex(mapping => new { mapping.TargetDataSetVersionId })
                 .HasDatabaseName("IX_DataSetVersionMappings_TargetDataSetVersionId")
                 .IsUnique();
 
-            builder.OwnsOne(v => v.LocationMappingPlan, locations =>
-            {
-                locations.ToJson();
-                locations.OwnsMany(l => l.Levels, locationMappings =>
-                {
-                    locationMappings.OwnsMany(mapping => mapping.Mappings, locationMapping =>
-                    {
-                        locationMapping.OwnsOne(lm => lm.Source);
-                        locationMapping.Property(lm => lm.Type)
-                            .HasConversion(new EnumToEnumValueConverter<MappingType>());
-                    });
-                    locationMappings.OwnsMany(mapping => mapping.Candidates);
-                });
-            });
-
-            builder.OwnsOne(mapping => mapping.FilterMappingPlan, filters =>
-            {
-                filters.ToJson();
-
-                filters.OwnsMany(f => f.Mappings, filterMapping =>
-                {
-                    filterMapping.OwnsOne(mapping => mapping.Source);
-                    filterMapping.Property(mapping => mapping.Type)
-                        .HasConversion(new EnumToEnumValueConverter<MappingType>());
-
-                    filterMapping.OwnsMany(mapping => mapping.OptionMappings, filterOptionMapping =>
-                    {
-                        filterOptionMapping.OwnsOne(mapping => mapping.Source);
-                        filterOptionMapping.Property(mapping => mapping.Type)
-                            .HasConversion(new EnumToEnumValueConverter<MappingType>());
-                    });
-                });
-
-                filters.OwnsMany(f => f.Candidates, filterTarget =>
-                {
-                    filterTarget.OwnsMany(mapping => mapping.Options);
-                });
-            });
+            builder.Property(p => p.LocationMappingPlan)
+                .HasColumnType("jsonb")
+                .HasConversion(
+                    v => JsonSerializer.Serialize(v, new JsonSerializerOptions(JsonSerializerDefaults.General)),
+                    v => JsonSerializer.Deserialize<LocationMappingPlan>(v,
+                        new JsonSerializerOptions(JsonSerializerDefaults.General))!);
+
+            builder.Property(p => p.FilterMappingPlan)
+                .HasColumnType("jsonb")
+                .HasConversion(
+                    v => JsonSerializer.Serialize(v, new JsonSerializerOptions(JsonSerializerDefaults.General)),
+                    v => JsonSerializer.Deserialize<FilterMappingPlan>(v,
+                        new JsonSerializerOptions(JsonSerializerDefaults.General))!);
```

Comment:
Just wondering if we need to bother with the `new JsonSerializerOptions(JsonSerializerDefaults.General)` bits. Would it work if we removed the optional options parameter in each of the `JsonSerializer.Serialize(...)` and `JsonSerializer.Deserialize(...)` lines?

Reply:
Done, and it's working fine